src/cordova/plugin/OutlineAndroidLib/outline/src/test/java/org/outline/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package org.outline;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
```

Comment:
Can we perhaps add a real test, even if simple?

Reply:
OK, I've added some minimal tests to exercise the test functionality.